openrtb2/bid.go

```diff
@@ -104,6 +105,23 @@ type Bid struct {
 	//   Substitution macros (Section 4.4) may be included.
 	AdM string `json:"adm,omitempty"`
 
+	// Attribute:
+	//   adm_native
+	// Type:
+	//   object
+	// Description:
+	//   Optional response for BidSwitch Native
+	//	 https://protocol.bidswitch.com/ssp-protocol/ssp-response-native-object.html
+	AdMNative native1.Response `json:"adm_native,omitempty"`
```

Comment:
These are not OpenRTB 2.6 fields. This package implements the [official standard](https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/2.6.md) and does not accept vendor specific changes.

Reply:
Sorry this was a mishap on our pulls, thanks for taking the time!